lib/oli_web/plugs/community_admin.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Oli.Plugs.CommunityAdmin do
+  alias Oli.Accounts
+  alias OliWeb.Router.Helpers, as: Routes
+
+  def init(opts), do: opts
+
+  def call(conn, _opts) do
+    if Accounts.is_admin?(conn.assigns.current_author) or
+         Plug.Conn.get_session(conn, :is_community_admin) do
+      conn
+    else
+      conn
+      |> Phoenix.Controller.put_flash(:info, "You are not allowed to see this page")
```

Comment:
This seems like a very developer message - is this consistent with other places in torus? :)